build-tools/src/main/resources/build-tools/spotbugs-filter.xml

```diff
@@ -90,4 +90,14 @@ limitations under the License.
     <Class name="org.apache.knox.gateway.audit.log4j.correlation.Log4jCorrelationService" />
     <Bug pattern="OBJECT_DESERIALIZATION" />
   </Match>
+
+  <Match>
+    <Class name="org.apache.knox.gateway.services.security.impl.ConfigurableEncryptor" />
+    <Bug pattern="DMI_RANDOM_USED_ONLY_ONCE" />
```

Comment:
I think this is a real finding.

https://github.com/apache/knox/blob/master/gateway-spi/src/main/java/org/apache/knox/gateway/services/security/impl/ConfigurableEncryptor.java#L106

that random could be created once instead of every time for encrypting bytes.